app/src/main/graphql/thirdPartyEvents.graphql

```diff
@@ -1,6 +1,6 @@
-mutation triggerThirdPartyEvent  ($triggerThirdPartyEventInput: TriggerThirdPartyEventInput!) {
-    triggerThirdPartyEvent(input: $triggerThirdPartyEventInput) {
-        message
-        success
-    }
-}
\ No newline at end of file
+#mutation triggerThirdPartyEvent  ($triggerThirdPartyEventInput: TriggerThirdPartyEventInput!) {
```

Comment:
commented out, but will be fixed in ticket -> https://kickstarter.atlassian.net/browse/MBL-1910